tools/build_langserver/outline.txt

```diff
@@ -78,3 +78,10 @@ init.go
 - handles initialization of the request
 
 
+Wishlist
+--------
+
+ - Autoformatting
+ - Completion of function argument names (e.g. `vis` -> `visibility`)
+ - Find references (`plz query revdeps` probably does most of this)
+ - rename support
```

Comment:
What's rename support?

Oh! you mentioned we could auto complete the `src` for local file names¬

Reply:
The `textDocument/rename` method, basically. Rename a build target & have all references across the project update.